lib/hooks/useEvent.lua

```diff
@@ -1,6 +1,8 @@
 local topoRuntime = require(script.Parent.Parent.topoRuntime)
 local Queue = require(script.Parent.Parent.Queue)
 
+local EVENT_CONNECT_METHODS = {"connect", "Connect", "on"}
```

Comment:
We should start with `Connect` here, otherwise all Roblox events will get connected to via `connect`, the deprecated one!

Reply:
That's true, we should prioritize the `Connect` method for the 'best' expected behavior!